Misc/NEWS.d/next/Library/2019-08-11-10-34-19.bpo-37819.LVJls-.rst

```diff
@@ -0,0 +1,2 @@
+Add Fraction.as_integer_ratio() to match the corresponding methods in bool,
```

Comment:
Would it be helpful to provide a link to the method using inline markup? This would allow readers to easily find the documentation for the new method.

Also regarding this line, as far as I'm aware, there's not a bool.as_integer_ratio() method. To confirm this, I used git grep to search the documentation:

![image](https://user-images.githubusercontent.com/44193521/62837949-a6df6600-bc43-11e9-9307-06a4a30f1b40.png)

As expected, it returned the methods for decimal, int, and float but nothing for bool.

```suggestion
Add :meth:`Fraction.as_integer_ratio` to match the corresponding methods in
```

Reply:
We don't have to provide cross-links from Misc/NEWS.  It is okay for entries there to be plain text.

Reply:
@rhettinger Oh yeah it's okay either way, but since the Misc/NEWS entries now support reST and Sphinx I think it's helpful to include links to functions, methods, or classes that were modified in functionality or documentation. Usually the news entries are very brief, so if a curious reader wants to get more information they can simply use the link.

To those of us who are familiar with the structuring of the documentation, it's usually easy to find the relevant sections. But, some less familiar or newer readers might have trouble doing so, especially if they use an external search engine (which frequently will lead them to the 2.7 docs or an incorrect page).

It's more a convenience/QoL addition for the readers than anything, and I don't think it hurts the readability or takes much effort from our end. I might start a thread over on python-dev to see what the others think. Not at all for this PR specifically, but just in general.

Reply:
Also, what was the conclusion on `bool` being in the news entry? From my understanding, there is not a `bool.as_integer_ratio()` function.

Reply:
@aeros167 Have you validated your understanding?

From my understanding, bool inherits from int.  If int has `as_integer_ratio()`, bool should have it too.

```
$ ./python
Python 3.9.0a0 (heads/master:e95ac20103, Jul 24 2019, 21:48:01)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> i = 42
>>> i.as_integer_ratio()
(42, 1)
>>> True.as_integer_ratio()
(1, 1)
```

Reply:
@methane Thanks for the explanation!

At first this didn't make sense to me, but then I remembered that:

```python3
>>> int(True)
1
>>> int(False)
0
>>> True == 1
True
>>> False == 0
True
```

So it makes sense that the boolean values would also have the int methods, since they are effectively specialized ints. I was just confused at first since I could not find an explicit mention of ``bool.is_integer_ratio()`` in the docs, but I suppose it doesn't need to be mentioned since as you said, bool inherits from int.